src/app/app.module.ts

```diff
@@ -90,7 +90,7 @@ const socialAuthServiceConfig = {
             },
             {
                 id: FacebookLoginProvider.PROVIDER_ID,
-                provider: new FacebookLoginProvider('293264554869933'),
+                provider: new FacebookLoginProvider('611379136173563'),
```

Comment:
google how Angular utilise environment variables and set Google and Facebook App IDs there. We don't have any example here, but look over the Internet how to do it.